### ISSUE TITLE
Client UI Design

### ISSUE THREAD

**Issue by SHU-Callum** [SmartMeterProject2024/Project#35]
Create the screen(s) for the Smart Meter desktop application, displaying data, buttons, and communicates error messages. User experience and accessibility should be considered

- [x] #52
- [x] #19 
- [x] #20 
- [ ] #36
